winjob/winjob.py

```diff
@@ -589,6 +589,6 @@ def read_task(file_data):
             try:
                 task = BinaryScheduledTask(file_data)
             except FormatError:
-                print "File is not XML or Binary job format"
+                print("File is not XML or Binary job format")
                 sys.exit(-1)
-        print json.dumps(task.parse(), indent=2)
\ No newline at end of file
+        print(json.dumps(task.parse(), indent=2))
```

Comment:
I'm not exactly sure if this code is supposed to run with Python 2 as well (the badge on the repo says 2.7) but this syntax will not work for Python 2 without an extra import added on the first line of this file.

```python
from __future__ import print_function
```

Example:

```python
Python 2.7.15rc1 (default, Apr 15 2018, 21:51:34) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> print(json.dumps("{}"), indent=2)
  File "<stdin>", line 1
    print(json.dumps("{}"), indent=2)
                                  ^
SyntaxError: invalid syntax
>>> 

```